python/cudf_polars/cudf_polars/dsl/expressions/string.py

```diff
@@ -222,6 +223,23 @@ def do_evaluate(
                     plc.Scalar.from_py(None, plc.DataType(plc.TypeId.STRING)),
                 )
             )
+        elif self.name is StringFunction.Name.ZFill:
+            child, width = self.children
+            assert isinstance(width, Literal)
+            try:
+                pa.scalar(width.value, type=pa.uint64())
```

Comment:
```suggestion
                plc.Scalar.from_py(width.value, dtype=plc.DataType(plc.TypeId.UINT64))
```

(We're trying to move away from using pyarrow)